docs/src/_concepts/state_backends.md

```diff
@@ -0,0 +1,95 @@
+for Meltano projects running in ephemeral environments or in circumstances where administering a dedicated backend database is undesirable, Meltano also supports persisting state in alternative "state backends". 
+
+## Installation
+
+To cut down on extraneous dependencies, Meltano imports cloud storage client libraries lazily and these libraries are not included in the base `meltano` package. 
+
+Only the default system database state backend _or_ the local filesystem state backend can be used as part of the base `meltano` distribution.
+
+To use a cloud storage backend, install Meltano using one of the following (extras)[https://peps.python.org/pep-0508/#extras]:
+- `meltano[aws]` to use the AWS S3 state backend.
+- `meltano[azure]` to use the Azure Blob Storage state backend.
+- `meltano[gcs]` to use the Google Cloud Storage state backend.
+
+## Configuration
+
+The state backend settings for your Meltano project can be configured much the same as any other Meltano setting. For some state backends, the only required setting is `state_backend.uri`. 
+By default, `state_backend.uri` is set to the special keyword string `systemdb`, which will tell Meltano to store state in the same backend database as it stores other project data. 
+
+It can be changed by running `meltano config meltano set state_backend.uri <URI for desired state backend>` or by directly editing a project's `meltano.yml` to add the following:
+```yaml
+state_backend:
+	uri: <URI for desired state backend>
+```
+For some state backends and in some environments, `uri` is the only setting that needs to be configured in order to use a state backend other than the default `systemdb` backend. For instance, to store state directly on the local filesystem at the path `${MELTANO_ROOT_DIR}/.meltano/state`, simply run `meltano config meltano set state_backend.uri file:///${MELTANO_ROOT_DIR}/.meltano/state`. 
+Then Meltano will store the state for a given `state_id` at the path `${MELTANO_ROOT_DIR}/.meltano/state/<state_id>/state.json`.
+
+### Locking
+
+Because the `systemdb` state backend utilizes a transactional database, it can rely on the database's transaction logic to prevent conflicts among multiple runs of the same pipeline.
+For some other state backends, Meltano implements its own simple locking mechanism.
+This locking mechanism utilizes reasonable defaults but you may wish to configure the locking settings differently. 
+You can do this via two additional top-level `state_backend` settings: `state_backend.lock_timeout_seconds` and `state_backend.lock_retry_seconds`.
+
+When Meltano tries to read or write state for a given `state_id`, it will try to acquire a _lock_ on the state data for that `state_id`.
+For example, using the local filesystem state backend with `state_backend.uri` set to `file:///${MELTANO_ROOT_DIR}/.meltano/state`, it will check to see if a file exists at the path `${MELTANO_ROOT_DIR}/.meltano/state/<state_id>/lock`. 
```

Comment:
```suggestion
For example, using the local filesystem state backend with `state_backend.uri` set to `file:///${MELTANO_PROJECT_ROOT}/.meltano/state`, it will check to see if a file exists at the path `${MELTANO_PROJECT_ROOT}/.meltano/state/<state_id>/lock`. 
```